samples/javascript/local.settings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "AzureWebJobsDashboard": "UseDevelopmentStorage=true",
+    "AZURE_STORAGE_CONNECTION_STRING": "<Put your Azure Storage connection string here>",
```

Comment:
What is the `AZURE_STORAGE_CONNECTION_STRING` setting?

Reply:
In order to write to blob storage I'm using the [Azure Storage SDK for Node](https://github.com/Azure/azure-storage-node). AZURE_STORAGE_CONNECTION_STRING is used by that module.

Reply:
Cool. My suggestion then is that we use the same connection string placeholder text for all three storage app settings. Using a non-emulator connection string is probably better since that's what non-Windows developers are most likely to require.